DEVELOPMENT.md

```diff
@@ -77,15 +77,30 @@ real life testing.
 
 ## Publishing a new kured release
 
-This is the most straight-forward in the whole process.
+Check that `README.md` has an updated compatibility matrix and that the
+url in the `kubectl` incantation (under "Installation") is updated to the
+new version you want to release.
+
+Now create the `kured-<release>-dockerhub.yaml`:
+
+```sh
+VERSION=1.3.0
+MANIFEST="kured-$VERSION-dockerhub.yaml"
+cat kured-rbac.yaml > "$MANIFEST"
+echo "---" >> "$MANIFEST"
```

Comment:
Not sure this is actually required, following this resulted in a:

```yaml
<snip>
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  namespace: kube-system
  name: kured
subjects:
- kind: ServiceAccount
  namespace: kube-system
  name: kured
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: kured
---
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: kured
  namespace: kube-system
<snip>
```

Reply:
Hah, yes, you're right - fixing!